Control coverage/Feature/BatchAccounts.md

```diff
@@ -0,0 +1,79 @@
+# Batch Accounts
+
+**Resource Type:** Microsoft.Batch/batchAccounts
+
+
+<!-- TOC -->
+
+- [Azure_BatchAccounts_Audit_Enable_Diagnostic_Settings](#azure_batchaccounts_audit_enable_diagnostic_settings)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_BatchAccounts_Audit_Enable_Diagnostic_Settings
+ 
+
+### Display Name 
+Enable Security Logging in Azure Batch Accounts
+
+### Rationale 
+Diagnostic logs must be enabled as they provide details for investigation in case of a security breach for threats.
+
+### Control Settings 
+```json 
+{
+    "DiagnosticForeverRetentionValue": "0",
+    "DiagnosticMinRetentionPeriod": "90",
+    "DiagnosticLogs": [
+        "ServiceLogs",
+        "ServiceLog",
+        "AuditLog"
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> Diagnostic setting meet the following conditions:
+>   1. Diagnostic logs are enabled.
+>   2. At least one of the below setting configured:
```

Comment:
Please update this at all place, its not setting, call it as log destination,

Reply:
Done.